src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -50,10 +50,18 @@
 
 
     <Target Name="NpmRunBuildHomePage" BeforeTargets="BeforeBuild">
+        <PropertyGroup>
```

Comment:
@aaron this change is coming https://github.com/DynamoDS/Dynamo/pull/15316 regarding the skip install step but it is only added for this npm target. Is this ok?

Reply:
Removed it.